Makefile

```diff
@@ -0,0 +1,60 @@
+.EXPORT_ALL_VARIABLES:
+COMMIT_HASH := $(shell git rev-parse HEAD)
+TIMESTAMP := $(shell date +%s)
+VERSION ?= $(shell git describe --tags --exact-match `git rev-parse HEAD` 2>/dev/null || echo 0.0.$(TIMESTAMP)-custom.$(COMMIT_HASH))
+TILE_NAME ?= $(shell if [ `echo $(VERSION) | grep -o custom` ]; then echo stackdriver-nozzle-custom; else echo stackdriver-nozzle; fi)
+TILE_LABEL ?= $(shell if [ `echo $(VERSION) | grep -o custom` ]; then echo "Stackdriver Nozzle (custom build)"; else echo Stackdriver Nozzle; fi)
+TILE_FILENAME := $(TILE_NAME)-$(VERSION).pivotal
+TILE_SHA256 := $(TILE_FILENAME).sha256
+RELEASE_TARBALL := stackdriver-tools-release-$(VERSION).tar.gz
+RELEASE_SHA256 := $(RELEASE_TARBALL).sha256
+RELEASE_BUILD_DIR := build
+TILE_BUILD_DIR := product
+RELEASE_PATH := $(RELEASE_BUILD_DIR)/$(RELEASE_TARBALL)
+
+build:
+	go build -v ./...
+
+test:
+	go test -v ./...
+
+lint:
+	# Tests for output
+	# Disabling gosec for https://github.com/securego/gosec/issues/267
+	# Disabling vetshadow for https://github.com/golang/go/issues/19490
+	# Disabling maligned because it also affect the config struct. TODO(mattysweeps) re-enable maligned
+	# Ignoring missing comments for now TODO(mattysweeps) fix godoc
+	[ -z "$$($(GOPATH)/bin/gometalinter --deadline=300s --disable gosec --disable vetshadow --disable maligned --vendor ./... | grep -v exported)" ]
+
+get-deps:
+	# For gometalinter linting
+	cd $(GOPATH) && wget -q -O - https://git.io/vp6lP | sh
```

Comment:
Installing the linter by running a shell exec with wget like this raises alarm bells. Is there a less sketchy way of doing this?

Reply:
Also, pulling the latest each time could potentially set yourself for a break down the line if they push a breaking change. Suggest instead downloading one of their releases and installing yourself: https://github.com/alecthomas/gometalinter/releases

Reply:
I pinned both the installer script and the linter to a specific version in the Makefile.
I updated the URL to be the original github link, instead of the obtuse git.io one.

I still like using the installer script for now because it works for all platforms (downloads the correct binaries)